### ISSUE TITLE
Unknown binaryTarget on Fedora37

### ISSUE THREAD

**Issue by pajasry** [prisma/prisma#12424]
### Bug description

Cannot generate prisma schema using `npx prisma generate` on Fedora 37. Fails with: `Error: Unknown binaryTarget rhel-openssl-3.0.x and no custom binaries were provided
`

### How to reproduce

1. run `npx prisma init`
2. run `npx prisma generate`

### Expected behavior

Prisma generates everything needed. (successfull execution)

### Prisma information

```datasource db {
  provider = "sqlite"
  url      = env("DATABASE_URL")
}

generator client {
  provider = "prisma-client-js"
}

model User {
  id    String @id @default(cuid())
  email String @unique

  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt

  password Password?
  notes    Note[]
}

model Password {
  hash String

  user   User   @relation(fields: [userId], references: [id], onDelete: Cascade, onUpdate: Cascade)
  userId String @unique
}

model Note {
  id    String @id @default(cuid())
  title String
  body  String

  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt

  user   User   @relation(fields: [userId], references: [id], onDelete: Cascade, onUpdate: Cascade)
  userId String
}```


### Environment & setup

- OS: Fedora 37
- Database: SQLite
- Node.js version: 16.13.0


### Prisma Version

Cannot run `prisma -v` because this error.

**Comment by janpio** [comment]
Related: https://github.com/prisma/prisma/issues/11356

**Comment by dpetrick** [comment]
Support for OpenSSL landed with 3.13, this is fixed.

**Comment by janpio** [comment]
Can you confirm @pajasry? Thanks.
